### PR TITLE
MGMT-15342: Adds pkg/graph unit-tests

### PR DESCRIPTION
**Pull request by nmagnezi**

### CODE REVIEW

**Comment by nmagnezi** [comment]
/hold
testing locally

**Comment by nmagnezi** [comment]
Tested okay:
```shell
INFO Successfully downloaded CoreOS ISO
INFO Successfully generated recovery CoreOS ISO
INFO Successfully pulled container registry image
INFO Successfully pulled OpenShift 4.12.12 release images required for bootstrap
INFO Successfully pulled OpenShift 4.12.12 release images required for installation
INFO Successfully generated data ISO
INFO Successfully downloaded appliance base disk image
INFO Successfully extracted appliance base disk image
INFO Successfully generated appliance disk image
INFO Time elapsed: 8m5s
INFO
INFO Appliance disk image was successfully created in assets directory: assets/appliance.raw
INFO
INFO Create configuration ISO using: openshift-install agent create config-image
```

File: pkg/graph/graph_test.go [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+}
+
+var _ = Describe("Test Graph", func() {
+	var (
+		testGraph Graph
+	)
+
+	It("GetReleaseImage - Valid version", func() {
+		version := "4.13.1"
+		graphConfig := GraphConfig{
```
Comment (danielerez) [inline pkg/graph/graph_test.go:75]: nit: we can init the GraphConfig in BeforeEach with common values. Then, on tests, just set for example graphConfig.Version, graphConfig.CincinnatiAddress, etc.

Comment (nmagnezi) [reply pkg/graph/graph_test.go:75]: Each test has its own `graphConfig`, so  It can't be done using `BeforeEach`

**Pull request merged**